test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -260,6 +300,25 @@ public FormLoginRequestBuilder acceptMediaType(MediaType acceptMediaType) {
 
 		private FormLoginRequestBuilder() {
 		}
+
```

Comment:
Let's make these public methods above the private constructor to follow Spring Security's conventions